injectedJavaScript/application.js

```diff
@@ -35,6 +35,7 @@ var content = (penColor, backgroundColor) => `
       }; */
       signaturePad.minWidth = 1;
       signaturePad.maxWidth = 4;
+      signaturePad.fromDataURL('${dataURL}');
```

Comment:
What do you think about adding a check to ensure that dataURL is not null/empty such that the signaturePad doesn't unnecessarily create an Image element with a potentially invalid source? (See https://github.com/szimek/signature_pad/blob/44423d11875dac43bc6daa16aed0f712f0bd928a/src/signature_pad.js#L87 )